internal/codegen/golang/templates/pgx/interfaceCode.tmpl

```diff
@@ -2,10 +2,10 @@
     type Querier interface {
     {{- range .GoQueries}}
         {{- if eq .Cmd ":one"}}
-            {{.MethodName}}(ctx context.Context, {{.Arg.Pair}}) ({{.Ret.Type}}, error)
+            {{.MethodName}}(ctx context.Context, {{.Arg.Pair}}) ({{if .Ret.IsPointer }}*{{end}}{{.Ret.Type}}, error)
```

Comment:
Instead of updating every reference to `.Ret.Type` and `.Ret.Name`, we should add a new method that takes into account `IsPointer` and returns the correct value.